common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -67,6 +67,18 @@ describe 'Problem', ->
     it 'bind the show button', ->
       expect($('div.action button.show')).toHandleWith 'click', @problem.show
 
+    it 'bind the display show answer tooltip', ->
+      expect($('div.action button.show')).toHandleWith 'focus', @problem.displayShowAnswerTooltip
```

Comment:
Nit: Despite the pre-existing tests, we generally prefer to just use class names instead of HTML elements in the selectors, to make it easier to modify the HTML in the future (and because the type of HTML element generally doesn't matter). That being said, for the button I can see keeping the button element in.

For example,  either expect($('.action button.show')).... or expect($('.action .show'))....